src/main/groovy/io/gatling/gradle/GatlingRunTask.groovy

```diff
@@ -51,6 +51,8 @@ class GatlingRunTask extends DefaultTask implements JvmConfigurable {
     void gatlingRun() {
         def gatlingExt = project.extensions.getByType(GatlingPluginExtension)
 
+        def logbackFile = new File(project.buildDir, "generated/gatlingLogback/logback.xml")
```

Comment:
* Duplicated code, file should be resolved with `LogbackConfigTask#logbackFile`
* Maybe move resolution close to usage, line 64

Reply:
Done